### ISSUE TITLE
Issue with naming/numbering convention

### ISSUE THREAD

**Issue by sjetha** [aspnet/EntityFramework6#834]
My concern with the release of this for .NET Core is that new users will not fully understand the reasons behind why there is an EF and an EF Core version available for .NET Core. It needs to be clearly stated somewhere that EF 6.3 is meant only as a tool for migrating from .NET Framework, and that EF Core is the version that should be used. The fact that EF's version number is higher than EF Core's won't help any new people make the right decision.

**Comment by ajcvickers** [comment]
@sjetha Agreed, but most of that naming and version numbers chosen are not something that we have much influence over. /cc @divega for the docs aspect.

**Comment by divega** [comment]
I have created https://github.com/aspnet/EntityFramework.Docs/issues/1540. If anyone has any suggestions on how to further improve the guidance, feel free to comment there or create a new documentation issue. 

For versioning we try to stick to semantic versioning as much as possible, although we do allow gaps to keep version numbers aligned with things like .NET Core and ASP.NET Core. After we jumped from EF v1 to EF v4, we voluntarily gave up the freedom to jump to new version numbers for non-engineering reasons.
